{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py

```diff
@@ -1,8 +1,12 @@
 # Django settings for project project.
-from os import environ
-from django.core.exceptions import ImproperlyConfigured
+from environ import Env, Path
 import pathlib
+from django.core.exceptions import ImproperlyConfigured
 
+env = Env()
```

Comment:
@jschneier By default it looks at `.env` and if it can't find it, I believe it pulls from `ENV`. I tried it out in the marex-dev shell and it pulls out all the heroku env variables without having a `.env` file.